contracts/PensionFundRelease.sol

```diff
@@ -109,14 +123,20 @@ contract PensionFundRelease {
 
     // release the fund
     function releaseRoots() returns (uint releasedAmount){
-        require(isReleaseApproved());
-        require(now > firstPaymentTime);
-
-        releasedAmount = getPaymentAmount();
-        if(releasedAmount > 0){
-            firtPaymentReleased = true;
+        // Confirm validators have released funds
+        if( !isReleaseApproved() || now < paymentTime){
```

Comment:
AFAIK, `require` is a recommended way for inputs validation. If you will try to execute a transaction, what will fail with require, usually Mist and Parity will warn you about fail. So the user will not spend any gas anyway, and it will give him clear understanding what's going on.

IMHO, `require` is a better way of input validation, so let's use it instead of `if`.